src/bytes.rs

```diff
@@ -145,6 +145,42 @@ impl Bytes {
         }
     }
 
+    /// No cost BytesMut to Bytes converter
+    ///
+    /// # Safety
+    /// Make sure `bytes.kind()` is `KIND_VEC`
+    #[inline]
+    pub(crate) unsafe fn from_bytes_mut_vec(bytes: BytesMut, off: usize) -> Bytes {
+        // into_boxed_slice doesn't return a heap allocation for empty vectors,
```

Comment:
`into_boxed_slice` not used here anymore

Reply:
OK, I erase all comment